src/main/java/frc/robot/subsystems/intake/IntakeSubsystem.java

```diff
@@ -388,19 +374,38 @@ public Command Idle() {
         () -> {});
   }
 
+  private boolean checkSimIntake(double expectedSpeed) {
+    return Math.signum(expectedSpeed) == Math.signum(intakeMotor.get());
+  }
+
+  private boolean checkSimConveyor(double expectedSpeed) {
+    return Math.signum(expectedSpeed) == Math.signum(conveyorMotor.get());
+  }
+
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
     SmartDashboard.putNumber("Intake/Intake Motor Output", intakeMotor.get());
     SmartDashboard.putNumber("Intake/Pivot Motor Output", pivotMotor.get());
     SmartDashboard.putNumber("Intake/Conveyor Motor Output", conveyorMotor.get());
-    SmartDashboard.putNumber("Intake/Pivot Setpoint", pivotSetpoint.in(Degrees));
+    SmartDashboard.putNumber("Intake/Pivot Setpoint (Degrees)", pivotSetpoint.in(Degrees));
     SmartDashboard.putNumber(
         "Intake/Pivot Position (Degrees)", pivotMotor.getPosition().getValue().in(Degrees));
     SmartDashboard.putNumber(
-        "Intake/Absolute Encoder Position (Rotation)",
+        "Intake/Absolute Encoder Position (Degrees)",
         Rotations.of(throughBoreEncoder.get()).in(Degrees));
     SmartDashboard.putString("Intake/Intake State", intakeState.toString());
+    SmartDashboard.putNumber(
+        "Intake/Belt Velocity (RPM)",
+        RotationsPerSecond.of(conveyorMotor.getVelocity().getValueAsDouble()).in(RPM));
+
+    // Log TOF Sensors
+    for (int i = 2; i < 5; i++) {
```

Comment:
```suggestion
    for (int i  : new int[]{kSensor2ID, kSensor3ID, kSensor4ID}) {
```
Trying to avoid magic numbers.